### ISSUE TITLE
Can't move mouse cursor to second screen

### ISSUE THREAD

**Issue by MondQ** [asweigart/pyautogui#162]
Hi,
Whenever I try to use pyautogui.moveTo() and enter coordinates for my second screen, e.g. 3150, 475 on 2 x 1920X1080 monitors, the mouse cursor moves to the edge of screen one (1919, 475) and stops there.

**Comment by alih552** [comment]
Hi, try something else like... pyautogui.moveTo(1000,400)

**Comment by brackleian** [comment]
Similar problem on Linux.
Also pyautogui.moveTo always treats Y value as zero and moves the cursor to the top of the screen.
Example: pyautogui.moveTo(1000, 500) actually moves to (1000, 0)

**Comment by awa5114** [comment]
Any news on this?

**Comment by Amusesmile** [comment]
What about on Mac? We're having this issue in an application I develop. For instance trying to click on the second monitor with something like this (where the first monitor is 1440 pixels wide): 

pyautogui.click(x=1661, y=924)

**Comment by gamegoofs2** [comment]
I am having this same issue. ctype works but moveTo does not.

**Comment by xavierbaez** [comment]
I am having the same issue with Red Hat Enterprise Linux 8.2
Any update on this?

**Comment by kylefoley76** [comment]
@Amusesmile, did you get it to work yet for a Mac?

**Comment by Amusesmile** [comment]
> @Amusesmile, did you get it to work yet for a Mac?

Yes, in the hackiest way possible. Say you want to do pyautogui.click(x=1661, y=924)- that won't work because it's on the second monitor. You might try something like moveTo(x=1661, y=924) then click() with no arguments. That still won't work because moveTo won't go to the second monitor. What DOES work is doing an hscroll of zero in place of the moveTo. Somehow hscroll has no trouble getting to the second monitor. So for example: 

pyautogui.hscroll(0, x=1661, y=924)
pyautogui.click()

Not sure if my syntax or arguments are right because I'm no python guy but that's the crux of it. 

Cheers,

**Comment by Amusesmile** [comment]
> I am having the same issue with Red Hat Enterprise Linux 8.2
> Any update on this?

see my comment above about hscroll. Might work for you.

**Comment by Amusesmile** [comment]
> I am having this same issue. ctype works but moveTo does not.

see above

**Comment by fenchu** [comment]
5years on, Would love to see this feature fixed. I have no control on which monitor the programs pops up on.

**Comment by kkew3** [comment]
`pyautogui.moveTo` and friends work with a second monitor on my macOS 12.6.5 without hacks, although `pyautogui` still claims that it supports only one monitor in FAQ:

> Q: Does PyAutoGUI work on multi-monitor setups.
>
> A: No, right now PyAutoGUI only handles the primary monitor.
